live/src/main/java/org/dbpedia/extraction/live/queue/LiveQueue.java

```diff
@@ -23,7 +23,7 @@
 
     private static final PriorityBlockingQueue<LiveQueueItem> queue = new PriorityBlockingQueue<LiveQueueItem>(1000);
     // this is a Map<long,int>, long is the pageID and int the number of same items in the queue (see add())
-    private static final Set<Long> uniqueSet = new HashSet(1000);
+    private static final Set<String> uniqueSet = new HashSet(1000);
```

Comment:
We adjusted the `OAIFeederMappings` and the `UnmodifiedFeeder` to ensure every `LiveQueueItem` has a title. There isn't any use of the smaller `LiveQueueItem` constructor (without passing a title) any more for an object that is added to the `LiveQueue`.